src/engine/test/health_test/engine-health-test/src/health_test/load_decoders.py

```diff
@@ -41,7 +40,10 @@ def load_integrations(ruleset_path: Path, engine_handler: EngineHandler) -> None
         # Load integration
         print(f"Loading integration...\n")
         rs = ResourceHandler.ResourceHandler()
-        engine_integration_add(engine_handler.api_socket_path, ns, integration_dir.resolve().as_posix(), False, rs)
+        try:
+            engine_integration_add(engine_handler.api_socket_path, ns, integration_dir.resolve().as_posix(), False, rs, without_debug=True)
+        except Exception as e:
+            sys.exit(e)
```

Comment:
sys.exit raises an exception and the program [exists only if the exception is not handled](https://docs.python.org/3/library/sys.html#sys.exit). It's not idiomatic to use sys.exit in functions, only in main function.